### PR TITLE
Fix check test for strace

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Fix broken strace tests by patching expected results from upstream. The proper values come from the upstream source repo from these two commits.
https://github.com/strace/strace/commit/4377e3a1535a0ec3a42da8a1366ad6943f4efa0e
https://github.com/strace/strace/commit/a75c7c4bcb6b48680275de3e99e17e0ebec811ec

###### Change Log  

- Patch strace test file

###### Does this affect the toolchain?  

YES
**NO**

###### Test Methodology

- Local package with with RUN_TESTS=y

### CODE REVIEW

**Review by thcrain-msft — APPROVED** [review]
Changes look good- only thing I would add is some documentation as to where these new test values came from.

File: SPECS/strace/strace.spec [hunk 1]
```diff
@@ -51,9 +56,10 @@ rm -rf %{buildroot}/*
 %{_mandir}/man1/*
 
 %changelog
-* Sat May 09 00:21:09 PST 2020 Nick Samson <nisamson@microsoft.com> - 5.1-2
-- Added %%license line automatically
-
+*   Wed Jan 06 2021 Andrew Phelps <anphel@microsoft.com> 5.1-3
+-   Patch tests with expected results. Increase test timeout.
```
Comment (PawelWMS) [inline SPECS/strace/strace.spec:60]: Wasn't `strace` one of the flaky tests, which passed or failed randomly? I'm asking to make sure, that updating expected values is the right solution.

Comment (anphel31) [reply SPECS/strace/strace.spec:60]: yes, these tests were inconsistent and not always failing. I updated the summary of this PR with the source of the test value changes. I'm expecting the test timeout increase to help with reliability as well

**Comment by anphel31** [comment]
> Changes look good- only thing I would add is some documentation as to where these new test values came from.

Added links to original upstream commits in the summary.

**Pull request merged**